ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
No it wouldn't be better to reset to "jbossts-properties.xml" because this is just a test.

All I am saying is that after your changes the new code is semantically identical to:

`System.setProperty("com.arjuna.ats.arjuna.common.propertiesFile", "commitmarkableresourcejbossts-properties.xml");`

in which case you should use that since the changes you have introduced add needless complexity.

On the other hand, if you think your changes are not semantically identical to a single System.setProperty then please explain why (I may not have understood how your changes work)